fidesctl/src/fideslang/models.py

```diff
@@ -92,6 +92,7 @@ class DatasetField(BaseModel):
     data_qualifier: FidesKey = Field(
         default="aggregated.anonymized.unlinked_pseudonymized.pseudonymized.identified",
     )
+    fields: Optional[List["DatasetField"]]
```

Comment:
Had to dig back into why! I was going through the [pydantic self-reference documentation](https://pydantic-docs.helpmanual.io/usage/postponed_annotations/#self-referencing-models) and saw that it can be referenced as a string as the first option, so went with that vs referencing by type. Just tested out switching to the other method by type (adding `from __future__ import annotations`) and that works as well, so can switch that out 👍🏽